web/html-forms.md

```diff
@@ -0,0 +1,164 @@
+There are various elements which can be used in the form which are as follows:
+1. Input element : It is the most commonly used field in the form which allows the user to specify different types of values. It can be text field, number field, password field, checkbox etc.
+
+Following is an example which shows the code snippet to declare the input element.
+
+```
+<form>
+    <label for="name">Name:</label>
+    <input type="text" name="name" id="name">
+</form>
+```
+
+Label is an element which is used to provide the context of the input field. Like in the above example the input field is provided to enter the Name.
+Type is used to define the type of the input box. 
+Checkbox is a type of input field which allows the user to select options from the given choices. 
+Following is an example which shows the code snippet when the input type is checkbox.
+
+```
+<form>
+  <input type="checkbox" id="apple" name="apple" value="Apple">
+  <label for="apple">Apple</label><br>
+  <input type="checkbox" id="mango" name="mango" value="Mango">
+  <label for="mango">Mango</label><br>
+  <input type="checkbox" id="banana" name="banana" value="Banana">
+  <label for="banana">Banana</label>
+</form>
+```
+
+Another important type of input field is date. This field allows user to enter only date
+Following is an example which shows the code snippet when the input type is date.
+```
+<form>
+  <label for="dob">DOB:</label>
+  <input type="date" id="dob" name="dob">
+</form>
+```
+
+2. Select Element : It is used when we have to provide options to the user in the dropdown.
+
+Following is an example which shows the code snippet to declare the Select element.
+
+```
+<form>
+    <select id="fruits" name="fruits">
+    <option value="apple">Apple</option>
+    <option value="mango">Mango</option>
+    <option value="banana">Banana</option>
+    </select>
+</form>
+```
+
+3. Textarea Element : It can be seen as an input field only which allows text in multiple lines.
+
+Following is an example which shows the code snippet to declare the Textarea Element.
+
+```
+<form>
+    <label for="message">Message:</label>
+    <textarea rows="2" cols="30" name="message"  id="message"></textarea>
+</form>
+```
+
+4. Button Element : It is a clickable element which is usually used to submit the data.
+
+Following is an example which shows the code snippet to declare the Button Element.
+
+```
+<form>
+    <button type="button">Click Me!</button>
+</form>
+```
+
+#### Validation Attribute
+HTML elements which allows restriction in the form elements are the validation attribute.
+1. Readonly Attribute : This attribute do not allow any text to be entered in the input field.
+```
+<form>
+  <label for="name">Name:</label><br>
+  <input type="text" id="name" name="name" value="Shreya" readonly><br>
+  <label for="email">Email:</label><br>
+  <input type="email" id="email" name="email" value="shreyakapoor98@gmail.com">
```

Comment:
There is no need of these lines here. Please remove similar invalid lines from `Disabled Attribute` and `Size Attribute` sections also. 
Don't you think there are some more important validation attributes like `required`, `minlength`, `maxlength` etc. that too needs to be add?